share_app/src/main/java/org/odk/share/controller/WifiHotspot.java

```diff
@@ -24,9 +26,11 @@
     private WifiManager wifiManager;
     private WifiConfiguration lastConfig;
     private WifiConfiguration currConfig;
-    private static final String ssid = "ODK-Share";
+    private Context context;
+    private static final String ssid = "hotspot";
 
     public WifiHotspot(Context context) {
```

Comment:
Better name of this class would be something like WifiHotspotManager or WifiHotspotHelper